SQL/New_patches/2018_07_23_battery_manager_permissions.sql

```diff
@@ -1,7 +1,8 @@
 -- Add Battery Manager to LorisMenu
-INSERT INTO   LorisMenu (Parent, Label, Link, Visible, OrderNumber)
+INSERT INTO LorisMenu (Parent, Label, Link, Visible, OrderNumber)
        VALUES (
-           (SELECT ID FROM LorisMenu WHERE Label = 'Admin', 'Battery Manager'),
+           (SELECT ID FROM (SELECT * FROM LorisMenu) as lm WHERE Label = 'Admin'),
```

Comment:
... (SELECT ID, 'Battery Manager, null, 7 FROM LorisMenu where Label = 'Admin' );...

Reply:
Instead of `SELECT *`?

Reply:
instead of lines 4-9

Reply:
Just saw your updated comment.

Right, you can't insert in the same table your selecting from.

Reply:
SELECT ID FROM (SELECT ID FROM LorisMenu WHERE Label = 'Admin') AS LM, ...

Reply:
Sounds good!